.circleci/config.yml

```
@@ -182,16 +184,42 @@ commands:
+                "$HOME/miniconda/bin/conda" run -n samo-dl-stable pip install -e ".[test]" || true
+              fi
             else
+              set +e
               "$HOME/miniconda/bin/conda" env update -f environment.yml --prune
+              update_rc=$?
+              set -e
+              if [ $update_rc -ne 0 ]; then
+                echo "Conda env update failed (possibly channel 403). Continuing with existing env." >&2
+              fi
             fi
             
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)